src/App.js

```diff
@@ -0,0 +1,42 @@
+import './App.css';
+import React from 'react';
+import { Switch, Link, Route } from 'react-router-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+import AddRecipe from './components/AddRecipe';
+import RecipesList from './components/RecipesList';
+import Recipe from './components/Recipe';
+
+function App() {
+  return (
+    <div className="App">
+      <nav className="navbar navbar-expand navbar-dark bg-dark">
+        <a href="/recipe/" className="navbar-brand">
+          Recipes
+        </a>
+        <div className="navbar-nav mr-auto">
+          <li className="nav-item">
+            <Link to={"/recipes"} className="nav-link">
```

Comment:
Small nitpick, you don't need the curly braces here, you can just do `<Link to="/recipes" ...`

Reply:
Doh! It's still a good thing to clean up! Thank you!